package.json

```diff
@@ -103,7 +103,8 @@
     "turndown": "^5.0.3",
     "tween": "^0.9.0",
     "url-parse": "^1.4.7",
-    "xml2json": "^0.11.2"
```

Comment:
We use `yarn`, can you remove the npm `package-lock` file and run `yarn` and commit the `yarn.lock` changes please? :)